src/server.rs

```diff
@@ -414,11 +427,11 @@ async fn index(
     payload: web::Payload,
     const_router: web::Data<Arc<ConstRouter>>,
     middleware_router: web::Data<Arc<MiddlewareRouter>>,
-    global_request_headers: web::Data<Arc<Headers>>,
-    global_response_headers: web::Data<Arc<Headers>>,
+    global_headers: (web::Data<Arc<Headers>>, web::Data<Arc<Headers>>),
+    response_header_exclude_paths: web::Data<Option<Vec<String>>>,
     req: HttpRequest,
 ) -> impl Responder {
-    let mut request = Request::from_actix_request(&req, payload, &global_request_headers).await;
+    let mut request = Request::from_actix_request(&req, payload, &global_headers.0).await;
```

Comment:
I am not a big fan of a tuple style access here. please keep it to global_request_headers

Reply:
this was to fix ci: cargo clippy warning https://rust-lang.github.io/rust-clippy/master/index.html#too_many_arguments

any other suggestions to do so?